### PR TITLE
Add Look Details Sections

### PR DESCRIPTION
**Pull request by k-ankobia**
Includes test and views for Look info page. 
Setup dependencies for router

### CODE REVIEW

**Review by ghost — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -25,10 +25,13 @@
     "react-testing-library": "^6.0.0"
   },
   "dependencies": {
+    "normalize.css": "^8.0.1",
     "prop-types": "^15.7.2",
-    "react": "^16.8.4",
+    "react": ">=15",
```
Comment (kaihoffman) [inline package.json]: Will this cause an issue?

File: src/LookInfo/index.js [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+const LookDetails = () => {
+  return (
+    <div>
+      <Heading>Brands</Heading>
+      <Paragraph>Ralph Lauren</Paragraph>
+
+      <Heading>Description </Heading>
+      <Paragraph>
+        Lorem Ipsum is simply dummy text of the printing and typesetting
+        industry.Lorem Ipsum has been the industry 's standard dummy text ever
```
Comment (kaihoffman) [inline src/LookInfo/index.js:77]: Is this freaking out for anyone else?

**Review by kaihoffman — COMMENTED** [review]
Please see individual comments @k-ankobia @ibrahimbutt @JPProgrammingTime

File: package.json (hunk 1 above)
Comment (kaihoffman) [reply package.json]: Edited in subsequent push

**Review by kaihoffman — APPROVED** [review]

**Review by KrzysztofBalejko — APPROVED** [review]
Approved

**Pull request merged**